source/layouts/layout.slim

```diff
@@ -57,3 +57,9 @@ html.Theme itemscope='' itemtype='http://schema.org/WebPage' lang=data.site.lang
       try{Typekit.load();}catch(e){}
     = javascript_include_tag 'all'
     = yield_content :javascripts
+    - unless development?
```

Comment:
did you test this? I don't think `development?` is really a thing. @zamith wrote that as pseudocode I think

We've been using `if build?` so far
example: https://github.com/subvisual/subvisual.co/blob/b4d53f1221e367c9489299ed35490333234efe12/source/javascripts/_segment.js.erb